### PR TITLE
Format

### PR DESCRIPTION
**Pull request by JoshuaLampert**
Maybe it's a good idea to also format P4est.jl. I excluded the wrapped file for now.  I think most of it looks acceptable. What do you think, @sloede, @ranocha?

### CODE REVIEW

**Comment by sloede** [comment]
> Maybe it's a good idea to also format P4est.jl. I excluded the wrapped file for now. I think most of it looks acceptable. What do you think, @sloede, @ranocha?

I think formatting is a good idea. I wouldn't format the wrapped file, since it is auto-generated, which makes it hard to compare it against changes once we have to re-generate it for some reason.

**Comment by JoshuaLampert** [comment]
> I think formatting is a good idea. I wouldn't format the wrapped file, since it is auto-generated, which makes it hard to compare it against changes once we have to re-generate it for some reason.

I think so, too. We could in principle format the file at the end of the generator, but would still advocate for not formatting the wrapped file.

**Comment by JoshuaLampert** [comment]
Again and again, we have a segfault when building the docs in CI. Do you know why?

File: .github/workflows/FormtatCheck.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      matrix:
+        julia-version: [1]
+        julia-arch: [x86]
+        os: [ubuntu-latest]
+    steps:
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: ${{ matrix.julia-version }}
+
+      - uses: actions/checkout@v4
```
Comment (ranocha) [inline .github/workflows/FormtatCheck.yml:23]: Maybe also add the Julia cache action here?

**Review by ranocha — APPROVED** [review]
I agree with what you have said above - good idea :+1:

**Comment by ranocha** [comment]
> Again and again, we have a segfault when building the docs in CI. Do you know why?

No idea, sorry. Looks like this has been happening for quite some time already...

File: .github/workflows/FormtatCheck.yml (hunk 1 above)
Comment (JoshuaLampert) [reply .github/workflows/FormtatCheck.yml]: Good idea. Done!

**Review by ranocha — APPROVED** [review]
Thanks! I didn't check every line in the modified files but I think it should be fine

**Comment by JoshuaLampert** [comment]
Are you fine with merging, @sloede?

**Review by sloede — APPROVED** [review]
I only did a quick scan of the changes to `test/`, but nothing overly atrocious has caught my attention. LGTM, thanks!

**Pull request merged**